.github/workflows/kustomize-build-ci.yaml

```diff
@@ -33,29 +33,12 @@ jobs:
         run: |
           cd kustomize/
           kubectl kustomize .
-      - name: kustomize build each component
+      - name: kustomize build tests
```

Comment:
The previous generic approach doesn't work anymore with such scenario where we have variable/token in Kustomize overlays.